draft-ietf-rtcweb-jsep.xml

```diff
@@ -723,18 +723,24 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               candidate gathering and maximize multiplexing, at the cost of less
               compatibility with legacy endpoints.
               </t>
-              <t></t>
-              <t hangText="max-bundle-and-rtcp-mux:">Similar to max-bundle,
-              but RTCP candidates are not gathered. This policy reduces the
-              candidates that must be gathered to the absolute minimum,
-              but will not be compatible with legacy endpoints that do not
-              support RTCP mux.
-              </t>
             </list>
             As it provides the best tradeoff between performance and
             compatibility with legacy endpoints, the default BUNDLE policy
             MUST be set to "balanced".
           </t>
+
+          <t>The
+          application can specify its preferred policy regarding use of RTCP-MUX
+          <xref target="RFC5761"/> using one of the following policies:
+          <list style="hanging">
+            <t hangText="compat:">The browser will gather both RTP and
```

Comment:
I changed to "max-compat" and "max-mux" which are more analogous but also more silly since this is a boolean.